add-chain/e2e_test.go

```diff
@@ -90,6 +90,24 @@ func TestAddChain_Main(t *testing.T) {
 			compareJsonFiles(t, "superchain/extra/genesis-system-configs/sepolia/", tt.name, tt.chainShortName)
 		})
 	}
+
+	t.Run("compress-genesis", func(t *testing.T) {
```

Comment:
This test runs as part of the `TestAddChain_Main` because its outputs are used as inputs to the `add-chain check-genesis-config` command/test that is added in https://github.com/ethereum-optimism/superchain-registry/pull/354